index.js

```diff
@@ -111,7 +111,7 @@ var vueTouchEvents = {
                     triggerEvent(event, this, 'tap')
                 }
 
-            } else if (!$this.swipeOutBounded) {
```

Comment:
I don't think this is a typo here. `touchMoveEvent` has set a value for `$this.swipeOutBounded`, so if `$this.swipeOutBounded` is true here, it will do nothing